README.md

```diff
@@ -30,6 +30,19 @@ This template will provide the full setup under the hood and will save a big por
   - [Storybook doesn't Support Some NextJS Features](#storybook-doesnt-support-some-nextjs-features)
   - [License](#license)
 
+## Deployment Guideline
+
+### How to Deploy on Vercel
+
+1. Create a vercel account on this [**link**](https://vercel.com/signup)
+2. Import your repository on this [**link**](https://vercel.com/import/git)
```

Comment:
```suggestion
1. Create a vercel account on from [**Vercel's Signup Page**](https://vercel.com/signup)
2. Import your repository on this [**Vercel's Dashboard**](https://vercel.com/import/git)
```

Reply:
@rafat17 could you please change this?

Reply:
@Showrin bhai, I have changed the reordering and rewording of a few steps under `How To Deploy on Vercel` section.
Thanks